pyproject.toml

```diff
@@ -76,9 +79,24 @@ test = [
 
 dev = ["tox", "pre-commit", "virtualenv", "pip", "twine", "toml", "bump2version"]
 
+[[tool.mypy.overrides]]
+module = [
+    "aiofiles",
+    "orjson",
+    "vdom",
+    "repr_llm.*",
+    "origami.*",
```

Comment:
Both of these are typed libraries -- perhaps there's more to configure to include the types in the shipped packages.